cmake/FindLua52.cmake

```diff
@@ -29,7 +29,7 @@
 find_path(LUA_INCLUDE_DIR lua.h
   HINTS
     ENV LUA_DIR
-  PATH_SUFFIXES include/lua52 include/lua5.2 include/lua-5.2 include/lua include
+  PATH_SUFFIXES include/lua52 include/lua5.2 include/lua-5.2 include include/lua51 include/lua5.1 include/lua-5.1
```

Comment:
Also @MoKob just reported on his OSX / brew setup:

> ls /usr/local/include/lua
> lua-5.1/    lua5.1/     luabind/    luajit-2.0/

will removing the `/include/lua` search path break this setup?